CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Changelog for AMD Debugger API (ROCdbgapi)
 
-Full documentatino for AMD Debugger API is available at
-[rocm.docs.amd.com](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/index.html).
+Full documentation for AMD Debugger API is available at
+[rocm.docs.amd.com/rocdbgapi](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/index.html).
 
 ## rocm-dbgapi-0.77.0
+
 ### Added
-- Added support for gfx1200 and gfx1201 architectures.
```

Comment:
All other items in the changelog have a final period at the end of the item.  Was that part of the change intended?